scss/detail.scss

```diff
@@ -1,18 +1,42 @@
 .book-container {
-  border: 1px solid $light-gray;
+  background-color: $white;
+  box-shadow: 6px 6px 5px rgba($black, 0.4);
+  display: flex;
+  left: 50%;
+  margin: 20px;
+  padding: 25px;
+  position: absolute;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  width: 800px;
+}
+
+.book-cover {
+  height: 340px;
+  width: 240px;
+}
+
+.book-information {
   display: flex;
   flex-direction: column;
-  height: 195px;
+  height: 220px;
   justify-content: space-between;
-  margin: 15px;
-  padding: 10px;
-  width: 300px;
+  padding: 0 40px;
 }
 
 .book-title {
   font-size: $text-big;
   font-weight: bold;
   padding-bottom: 30px;
+  &::after {
+    background-color: $light-green;
+    content: '';
+    height: 4px;
+    left: 38%;
+    position: absolute;
```

Comment:
if a child node has `position: absolute;` its parent should have `position: relative;`.

This [article](https://css-tricks.com/absolute-relative-fixed-positioining-how-do-they-differ/) can help you